tasks/windows.yml

```diff
@@ -6,11 +7,55 @@
     type: string
     state: present
 
-- name: changing hostname to {{ inventory_hostname }}
+# firewall configuration
```

Comment:
Do we actually need to use the host-based firewall?

Reply:
I think @juanpgomez-gsa was saying it checks a few compliancy boxes to have it enabled... We won't be doing anything with it so thats why I went all:all in/out to check those boxes.

Reply:
ok, well go with it for now we'll circle back on that with security

Reply:
👍